cfme/tests/services/test_provision_stack.py

```diff
@@ -148,15 +152,22 @@ def prepare_stack_data(provider, provisioning):
         }
     else:
         stack_prov = provisioning['stack_provisioning']
-
-        stack_data = {
-            'stack_name': stackname,
-            'stack_timeout': stack_timeout,
-            'vm_name': vm_name,
-            'key_name': stack_prov['key_name'],
-            'select_instance_type': stack_prov['instance_type'],
-            'ssh_location': provisioning['ssh_location']
-        }
+        if appliance.version < "5.8":
```

Comment:
This can be dropped down after the breakpoint tag was added for 57z:
https://github.com/ManageIQ/integration_tests/releases/tag/5.7-breakpoint

Reply:
Done